janitor/functions/utils.py

```diff
@@ -622,6 +622,27 @@ def get_index_labels(
     return index[_select_index(arg, df, axis)]
 
 
+def get_columns(group, label):
```

Comment:
Could we get type hints here, please? They're helpful when reading the function signatures.

```suggestion
def get_columns(group: Union[DataFrameGroupBy, SeriesGroupBy], label):
```